spec/dfn2toml.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+from collections.abc import Mapping
+from pathlib import Path
+from typing import Any
+
+from flopy.utils import import_optional_dependency
+
+from flopy4.dfn import Dfn
+
+SPEC_PATH = Path(__file__).parent
+DFN_PATH = SPEC_PATH / "dfn"
+TOML_PATH = SPEC_PATH / "toml"
+
+
+class Shim:
+    @staticmethod
+    def _attach_children(d: Any):
+        if isinstance(d, Mapping):
+            if "children" in d:
+                for n, c in d["children"].items():
```

Comment:
Does children still exist? I thought we got rid of that part by just having variables directly named in the toml.